Packs/PAN-OS/Integrations/Panorama/Panorama_test.py

```diff
@@ -6195,21 +6195,6 @@ def test_add_time_filter_to_query_parameter_fail(self):
             add_time_filter_to_query_parameter('time_generated geq "2020-01-01T00:00:00Z"',
                                                fetch_incidents_input.utc_time_twelve)
 
-    @pytest.mark.parametrize('query, id, expected_result',
-                             fetch_incidents_input.test_add_unique_id_filter_to_query_parameter_args)
-    def test_add_unique_id_filter_to_query_parameter(self, query, id, expected_result):
```

Comment:
Why this test was removed?

Reply:
The original function was removed, therre is no use anymore.